otrv4.md

```diff
@@ -979,8 +982,8 @@ User Profile (USER-PROF):
   Profile Expiration (PROF-EXP)
   Public Shared Prekey (ED448-SHARED-PREKEY)
     The shared prekey used between different prekey messages.
-  Profile Signature (EDDSA-SIG)
   (optional) Transitional Signature (SIG)
+  Profile Signature (EDDSA-SIG)
 ```
```

Comment:
The problem with this is actually the signing process. If versions 3 and 4 are supported, you sign `Ed448 public key || Versions || Profile Expiration || Public Shared Prekey` with OTRv3 DSA key and denote that value `Transitional Signature`.  Then you sign `m || Transition Signature` with the symmetric key, and denote this value `Profile Signature`.  So probably the order should be chacked.

Reply:
Something to check further might be that in some places we are calling `Transition Signature` and on others `Transitional Signature`. This should be consistent.

Reply:
I'll find and put. But What do you prefer?

Reply:
I think it should be an adjective, so transitional. What do you think?

Reply:
If this signature isn't used after this, I think that transitional is better

Reply:
Yeah, transitional seems nice :)